installer.sh

```diff
@@ -1,10 +1,9 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
-echo "This script is LINUX ONLY, and it's still in testing phases, if you encounter any bugs please open an issue"
+echo "This script is for LINUX and macOS, and it's still in testing phases. If you encounter any bugs, please open an issue."
```

Comment:
Can you change "LINUX" to "linux", it was capitalized because it was very important that you understood that it only supported linux before

Reply:
Changed